tests/test_modeling_tf_common.py

```diff
@@ -205,6 +205,46 @@ def test_save_load_config(self):
 
             self.assert_outputs_same(after_outputs, outputs)
 
+    def test_saved_model_creation(self):
```

Comment:
I added this test back into `test_modeling_tf_common` as: 

* Most models when testing took <30 sec
* Many models couldn't be saved with `saved_model=True` flag and their serving code wouldn't have worked. I think it's better to include this as default and skip if too long. This increases the chance of catching issues and makes the test more visible.